src/Umbraco.Core/Configuration/GlobalSettings.cs

```diff
@@ -395,6 +395,28 @@ public bool UseHttps
             }
         }
 
+        /// <summary>
+        /// Gets a value indicating whether the content dashboard should be available to all users.
+        /// </summary>
+        /// <value>
+        ///     <c>true</c> if the dashboard is visible for all user groups; otherwise, <c>false</c>
+        ///     and the default access rules for that dashboard will be in use.
+        /// </value>
+        public bool AllowContentDashboardAccessToAllUsers
```

Comment:
This method should just live on ContentDashboardSettings instead of GlobalSettings :)